clang/include/clang/Driver/SanitizerArgs.h

```diff
@@ -26,6 +26,8 @@ class SanitizerArgs {
   SanitizerSet RecoverableSanitizers;
   SanitizerSet TrapSanitizers;
   SanitizerSet MergeHandlers;
+  SanitizerSet TopHot;
+  SanitizerMaskWeights TopHotWeights = {0};
```

Comment:
It's not `Weight`, it's `cutoff threshold`.

Reply:
Renamed to Cutoff - is that ok instead of CutoffThreshold?